src/software/gui/drawing/ball.cpp

```diff
@@ -34,24 +34,29 @@ void drawBallVelocity(QGraphicsScene *scene, const Point &position,
 }
 
 void drawBallPosition(QGraphicsScene *scene, const Point &position,
-                      const double distance_from_ground, const QColor &color)
+                      const double distance_from_ground, QColor color)
 {
+    // A rough estimate of the max distance from the ground the ball will ever reach.
+    // This value sets the maximum above which no change will be visible.
+    const double ball_max_distance_from_ground = 1.25;
```

Comment:
pull this out into the header?

Reply:
although it's a constant this value is very specific to this function. Would putting it in the header be exposing it too publicly?

Reply:
make it static then?

Reply:
done